plugins/polio/js/src/domains/Notifications/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FunctionComponent } from 'react';
+import { Box } from '@material-ui/core';
+import { LoadingSpinner, useSafeIntl } from 'bluesquare-components';
+
+import TopBar from '../../../../../../hat/assets/js/apps/Iaso/components/nav/TopBarComponent';
+import { Router } from '../../../../../../hat/assets/js/apps/Iaso/types/general';
+
+import { useStyles } from '../../styles/theme';
+
+import MESSAGES from './messages';
+import { NotificationsFilters } from './Filters/NotificationsFilters';
+import { NotificationsParams } from './types';
+import { NotificationsTable } from './Table/NotificationsTable';
+import { getNotificationsDropdownsContent } from './hooks/api';
+
+type Props = { router: Router };
+
+export const Notifications: FunctionComponent<Props> = ({ router }) => {
+    const classes: Record<string, string> = useStyles();
+    const { data: dropdownContent, isFetching: isFetchingDropdownContent } =
+        getNotificationsDropdownsContent();
+    const { formatMessage } = useSafeIntl();
+    const { params } = router;
+    const paramsNew: NotificationsParams = {
+        ...params,
+        pageSize: params.pageSize || '20',
+        order: params.order || 'id',
+        page: params.page || '1',
+    };
+
+    return (
+        <>
+            {isFetchingDropdownContent && <LoadingSpinner />}
+            {!isFetchingDropdownContent && (
+                <>
+                    <TopBar
+                        title={formatMessage(MESSAGES.notificationsTitle)}
+                        displayBackButton={false}
+                    />
+                    <Box className={classes.containerFullHeightNoTabPadded}>
+                        <NotificationsFilters
+                            params={paramsNew}
+                            dropdownContent={dropdownContent}
```

Comment:
You've got a TS error here

Reply:
```suggestion
                            dropdownContent={dropdownContent as DropdownsContent}
```

Not blocking but the TS compiler is not smart enough to realize you won't pass the prop if `dropdownContent` is `undefined`